styles/style.css

```diff
@@ -0,0 +1,159 @@
+@import url('https://fonts.googleapis.com/css2?family=Tangerine:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Charm:wght@400;700&family=Tangerine:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Caudex:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+:root {
+
+    font-size: 16px;
+    /* CSS HEX */
+    --silver: #a5a19d;
+    --silver-2: #C6C0B6ff;
+    --chocolate-cosmos: #5B1C1Dff;
+    --field-drab: #65512Cff;
+    --white-smoke: #F3F3F3ff;
+
+}
+
+* {
+    /* border: 1px dotted blue; */
+}
+
+h1 {
+    font-family: "Tangerine", cursive;
+    font-weight: 400;
+    font-style: normal;
+}
+
+* {
+    font-family: "Caudex", serif;
+    font-weight: 400;
+    font-style: normal;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: var(--white-smoke);
+    color: #333;
+    height: 100vh;
+    display: grid;
+    grid-template-rows: 1fr 6fr .5fr;
+
+}
+
+header,
+footer {
+    background-color: var(--field-drab);
+    padding: 20px;
+    text-align: center;
+}
+
+header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    height: 150px;
+}
+
+.header-text-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+h1 {
+    font-size: 3rem;
+    font-weight: bold;
```

Comment:
There is already an `h1` selector block earlier in this file. I suggest having a single block per unique selector in each file so it's easy to see all styles applied to a specific selector in one place. 

Currently we have a situation where we have defined `font-weight` for the same selector in multiple blocks in the same file, so a reader has to do some extra thinking to determine what the outcome of the code should be.